config.yml

```diff
@@ -638,6 +643,9 @@ nodes:
         type: location?
       - name: closing_loc
         type: location?
+      - name: flags
```

Comment:
Can you make this the first field in the list? We just did this yesterday but we're making all of them the first.